vector/src/main/java/im/vector/app/core/extensions/BasicExtensions.kt

```diff
@@ -29,7 +30,7 @@ inline fun <T> T.ooi(block: (T) -> Unit): T = also(block)
 /**
  * Check if a CharSequence is an email.
  */
-fun CharSequence.isEmail() = Patterns.EMAIL_ADDRESS.matcher(this).matches()
+fun CharSequence.isEmail() = this.isEmail()
```

Comment:
Reading it from GitHub, it looks like an infinite loop (it's maybe one?).
Maybe remove this extension for clarity. The call site can import `org.matrix.android.sdk.api.extensions.isEmail` instead?

Reply:
Yes I removed this extension.